README.md

```diff
@@ -1,5 +1,20 @@
 # Regulatory R Package Repository
 
+Using R in submissions to healthcare regulators is challenging since the 
+quality of any packages used must be ensured and evidence of ensuring this
+quality must be made available on request. 
```

Comment:
I tried to more explicitly highlight the "why" and the desired outcomes.

```suggestion
Submissions to healthcare regulators rely on quality software for reliable, 
reproducible analysis, with evidence of this quality must be available on 
request.
To support recent trends in regulated industries toward adoption of open 
tooling, we aim to adapt these quality deliverables to the new ways that 
software is developed and distributed. It is our goal to embed these 
activities into shared resources for package distribution to make quality
more consistent and transparent.
```

Reply:
hmmm, this sounds a bit like we want to dev packages ourselves, isn't that rather the scope of the ASA WG on SWE https://rconsortium.github.io/asa-biop-swe-wg/ ?

Reply:
Hm - yeah, rereading it again I think there's still room for improvement. I'm not seeing the package development angle that you're talking about, but it definitely doesn't strike me as clear. Maybe we leave this out for now.